src/fluree/db/ledger/json_ld.cljc

```diff
@@ -90,6 +94,30 @@
         db*   (or db (ledger-proto/-db ledger (:branch opts*)))]
     (jld-commit/commit ledger db* opts*)))
 
+(defn default-context
+  "Returns default context at t (must be internal negative value)"
+  [ledger t]
+  (go-try
+    (let [latest-db (db ledger nil)
+          t*        (cond
+                      (string? t) (<? (time-travel/datetime->t latest-db t))
+                      (pos-int? t) (- t)
+                      (neg-int? t) t)
+          db-conn   (:conn latest-db)
+          flakes-ch (query-range/time-range latest-db :tspo = []
+                                            {:from-t t*, :to-t t*})
+          context   (json-ld/parse-context {:f "https://ns.flur.ee/ledger#"})
+          error-ch  (async/chan)
+          commits   (async/alt!
+                     (async/into [] (history/commit-flakes->json-ld
+                                     latest-db context error-ch flakes-ch))
+                     ([result] result)
+                     error-ch ([e] e))
+          _         (when (util/exception? commits) (throw commits))
+          [{:keys [f/commit]}] commits]
+      (->> commit :f/defaultContext :f/address
+           (jld-reify/load-default-context db-conn) <?))))
```

Comment:
tiny formatting nitpick: this is a little weird-looking to me with the `<?` where it is, maybe it could go on its own line? or everything in this `->>` on its own line?